structure.ts

```diff
@@ -0,0 +1,30 @@
+import { Steg, stegTittel, Ytelse, ytelseTittel } from './schemas/typer'
+import {
+  Divider,
+  ListItem,
+  ListItemBuilder,
+  StructureBuilder,
+  StructureContext,
+} from 'sanity/lib/exports/desk'
+
+export const structure = (S: StructureBuilder, _context: StructureContext) => {
+  const ytelseMappe = genererYtelseMappe(S)
```

Comment:
Gir denne navngivningen mening?

Reply:
`ytelseMappe` er jo egentlig det som genererer ytelsemappene, mens genererYtelseMappe lager funskjonen som genererer ytelsemapper. Hva med `ytelseMappe` -> `lagYtelseMappe` og `genererYtelseMappe`-> `lagYtelseMappeFunksjon`
```suggestion
  const lagYtelseMappe = genererYtelseMappeFunksjon(S)
```

Reply:
Og samme med `stegMappe` -> `lagStegmappe` og `genererStegMappe` -> `lagStegmappeFunksjon`